app/gateway/database/postgres/account/get.go

```diff
@@ -7,7 +7,19 @@ import (
 )
 
 func (r *repository) Get(ctx context.Context, id id.ExternalID) (account.Account, error) {
-	const query string = "select * from accounts where external_id = $1"
+	const query string = `select 
```

Comment:
```suggestion
	const query = `select 
```

Boa. Só não precisa especificar que é uma string